examples/complex/xss_scanner.py

```diff
@@ -0,0 +1,370 @@
+from mitmproxy import ctx
+from socket import gaierror, gethostbyname
+from urllib.parse import urlparse
+import requests
+import re
+from html.parser import HTMLParser
+from mitmproxy import http
+from typing import Dict, Union, Tuple, Optional, List, NamedTuple
```

Comment:
Now that this is in examples, let's add a brief comment here explaining what it does how to use it! :smiley: